spark/dl/src/main/scala/com/intel/analytics/bigdl/visualization/README.md

```diff
@@ -8,113 +8,39 @@
 >You can find the wheel repository at https://pypi.python.org/pypi/tensorboard
 
 
-Simply, with one pip command you can install it in Ubuntu. But it is very likely that you will run into problems while installing. If you meet any error, please check the instructions below.
+Simply, with one pip command you can install it in Ununtu. But it is very likely that you will run into problems while installing. If you meet any error, please check the instructions below.
 
-If you install it with Mac OS, similar in ubuntu, it is easy to install with pip. Notice that the wheels for mac do not require CPython to support wide unicode. If you meet any error, read the instructions below carefully. You may find your own solutions from these information. 
+If you install it with Mac OS, similar in ubuntu, it is easy to install with pip. Notice that the wheels for mac do not require CPython to support wide unicode. If you meet any error, read the instructions below carefully. You may find your own solutions from these information.
+
```

Comment:
1. Not tested yet.
2. It is that CPython which support wide unicode is not required on mac. Because the wheel for mac does not have this requiremennt according to the wheel tag: tensorboard-1.0.0a6-cp27-cp27m-macosx_10_11_x86_64.whl